### PR TITLE
extracts nice inspect to a module

### PR DESCRIPTION
**Pull request by Tuhaj**
Hey! I found that this class has a B instead of an A.
https://codeclimate.com/github/imanel/websocket-ruby/lib/websocket/frame/base.rb
How about extracting it to an module? Can it be done better than this way?

### CODE REVIEW

File: lib/websocket/handshake/base.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@ module Handshake
     # @abstract Subclass and override to implement custom handshakes
     class Base
       include ExceptionHandler
+      # Recreate inspect as #to_s will be overwritten
```
Comment (imanel) [inline lib/websocket/handshake/base.rb]: Could you move this comment to `inspect` implementation?

**Comment by imanel** [comment]
PS: Don't worry about Travis - looks like there were changes in RuboCop and new version is throwing some warnings.

File: lib/websocket/handshake/base.rb (hunk 1 above)
Comment (Tuhaj) [inline lib/websocket/handshake/base.rb]: Sure! Although I'm not sure if it is true. As I can see the `#to_s` method is not overwritten in the second case of the inclusion.

**Comment by Tuhaj** [comment]
Yup, rubocop throws:
> 68  files inspected, 55 offenses detected
Actually I could fix them with a next commit.

**Comment by imanel** [comment]
It's falling because we are still supporting [Ruby 1.9](https://github.com/jruby/jruby/issues/2771)

**Comment by Tuhaj** [comment]
@imanel all done 😄

**Pull request merged**